docs/modding/suitmodding/addinglocals.mdx

```diff
@@ -0,0 +1,666 @@
+```json
+     "en": {
+        "string1": "text you see in game",
+        "string2": "More text you see in game"
+     },
+     "fr": {
+     "string1": "francais text tu vieux en jeu",
+     "string2":"Psus text tu vieux en jeu"
+     }
+     ```
+    [[Return to 'Syntax']](#syntax)
+    
+    Example of a suit's name *line*:
+ ```json
+ "989928F2-06D6-42F3-871A-353F07DEF969_LEGACY_HERO_BLOODMONEYSUIT_M_HPA2293_NAME_": "Blood Money Suit"
+ ```
+ - The first part is the "string": `989928F2-06D6-42F3-871A-353F07DEF969_LEGACY_HERO_BLOODMONEYSUIT_M_HPA2293_NAME_`
+ - The second part will be the text you see in game: `Blood Money Suit`
+### 1. Find your Suit
+*To find the suit name you want to modify easily, you will have to enter search mode. To do that:* 
+
+- Click *inside* the <Tooltip text={<strong>"Preview box"</strong>} tooltip="A box that displays the contents of your LOCR file" /> and press `Ctrl + F`
+- type in your suit's name 
+:::warning Search Mode
+If you don’t click inside *Preview* box first, the search won’t work properly and may look for text elsewhere in  <Tooltip text={<strong>GlacierKit</strong>} tooltip="A modding tool used for editing and managing game files in Hitman: World of Assassination." />.
+>This feature is useful for other situations but doesn’t apply here.
+:::
+[[Return to Advanced Search]](#6-focus-on-the-right-file)
+ 
+### 2. Copy the string
+*You will only need the* <Tooltip text={<strong>string</strong>} tooltip= "A key that links to text in the LOCR File" /> *so copy it without the quotation marks!*
+
+Like so:
+`989928F2-06D6-42F3-871A-353F07DEF969_LEGACY_HERO_BLOODMONEYSUIT_M_HPA2293_NAME_`
+<div id="multilangua"></div>
+<details>
+<summary>Multi-language</summary>
+:::info multi-language
+Try pasting your copied <Tooltip text={<strong>string</strong>} tooltip= "A key that links to text in the LOCR File" /> in <Tooltip text={<strong>the search box</strong>} tooltip= "The one that appears when you search inside the 'Preview box'" />.
+
+You will see your <Tooltip text={<strong>string</strong>} tooltip= "A key that links to text in the LOCR File" /> attached to multiple languages of your suit's name. 
+This shows that a specific <Tooltip text={<strong>string</strong>} tooltip= "A key that links to text in the LOCR File" /> will always be the same in every other language `{array}` of a <Tooltip text={<strong>LOCR File</strong>} tooltip="A file that holds game text in different languages, linked by unique 'strings'." />. 
+>This is relevant for later!
+
+[[Return to Adding More Languages]](#5-optionally-add-more-languages)
+:::
+</details>
+
+
+---
+
+## Pt. 3: **CONVERT TO DECIMAL**
+*We need to convert the* <Tooltip text={<strong>string</strong>} tooltip= "A key that links to text in the LOCR File" /> *to a* <Tooltip text={<><strong>decimal number</strong><span className="icon-alert"></span></>} tooltip="A string converted into a numeral form" /> *to make it usable in the manifest file*.
```

Comment:
These tooltips are very duplicated, surely there's some better way of doing this in code

Reply:
There probably is but idk how really. Haven't seen a tooltip on any of the articles on the site and I'm new to coding. Probably the overuse of tooltips and using bold in general is the problem